UOP1_Project/Assets/Scripts/Characters/DropItem.cs

```diff
@@ -8,8 +8,12 @@ public class DropItem
 	Item _item;
 
 	[SerializeField]
-	float _dropRate;
+	float _itemDropRate;
+
+	[SerializeField]
+	private GameObject _collectibleItemPrefab;
```

Comment:
You are right! they are currently not populated in Item SO, because I guess the final version is not available yet. 
I will update the PR to rely on it and populate for the time being them with the collectible item mock prefab :)